Modules/ReportBuilder/Handlers/HeaderCompanyBlockHandler.php

```
@@ -0,0 +1,64 @@
+ *   "show_vat_id": true,
+ *   "show_phone": true,
+ *   "show_email": true,
+ *   "show_address": true,
+ *   "font_size": 10,
+ *   "font_weight": "bold",
+ *   "text_align": "left"
+ * }
+ */
+class HeaderCompanyBlockHandler implements BlockHandlerInterface
+{
+    public function render(BlockDTO $block, Invoice $invoice, Company $company): string
```

PHPMD (2.15.0)

Avoid unused parameters such as '$invoice'. (undefined)

(UnusedFormalParameter)